header.go

```diff
@@ -2593,7 +2638,7 @@ func (h *ResponseHeader) parseHeaders(buf []byte) (int, error) {
 					continue
 				}
 				if caseInsensitiveCompare(s.key, strTrailer) {
-					h.SetTrailerBytes(s.value)
+					_ = h.SetTrailerBytes(s.value)
```

Comment:
These functions return an error so we could return it here. It also returns an error on a bad content-length for example.